frontend/resourceadm/pages/ResourcePage/ResourcePage.tsx

```diff
@@ -176,14 +222,44 @@ export const ResourcePage = (): React.ReactNode => {
     return false;
   };
 
+  /**
+   * Gets the tabs to display. If showMigrate is true, the migration tab
+   * is added, otherwise it displays the three initial tabs.
+   *
+   * @returns the tabs to display in the LeftNavigationBar
+   */
+  const getTabs = (): LeftNavigationTab[] => {
+    if (getShowMigrate() && !leftNavigationTabs.includes(migrationTab)) {
+      return mapTabToIncludeMissingValues([...leftNavigationTabs, migrationTab]);
+    } else {
+      return mapTabToIncludeMissingValues(leftNavigationTabs);
+    }
+  };
+
+  /**
+   * Adds the onClick, isActive, and maps the tabName for each tab.
+   *
+   * @param tabs the tabs to map
+   *
+   * @return the mapped tabs
+   */
+  const mapTabToIncludeMissingValues = (tabs: LeftNavigationTab[]): LeftNavigationTab[] => {
+    return tabs.map((tab: LeftNavigationTab) => ({
+      ...tab,
+      onClick: (tabId: number) => navigateToPage(getPageByIndex(tabId)),
```

Comment:
Ref. de to tidligere kommentarene, så kunne denne vært forenklet til noe som dette:

`onClick: (tabId: NavigationBarPage) => navigateToPage(tabId),`

Reply:
Kunne kanskje vurdert å droppe tabId og brukte tabName direkte? 🤔

Reply:
Se kommentarene mine over 😄 
Var på den tanken med tabName da jeg laget komponenten, men så bruker tabName translation, og da tror jeg det kan bli litt krøll fort. Bedre å ha en unique ID som er enkel å bruke. 
![Screenshot 2023-09-13 at 08 03 33](https://github.com/Altinn/altinn-studio/assets/133344438/4f14cdd2-cd0a-4ff9-8fd1-4fd5d2301912)